modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1220,8 +1258,14 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 [%header,cols="30a,40a,30a"]
 |===
 |Command | Description |  Example
-|`--name` | New asset name .2+<.<| `exchange asset modify --name newName --tags test,sample`
-|`--tags` | Comma-separated tags for the asset
+|`--name` (required) | Asset name .2+<.<|
```

Comment:
This should be reverted.
The **only** change should be the addition of the `uploadv2`, no other command should be modified.

Reply:
Reverted